test/functional/Java16andUp/src/org/openj9/test/jep389/downcall/PrimitiveTypeTests.java

```diff
@@ -0,0 +1,588 @@
+		FunctionDescriptor fd = FunctionDescriptor.of(C_SHORT, C_SHORT, C_SHORT);
+		Symbol functionSymbol = nativeLib.lookup("add2Shorts").get();
+		MemoryAddress memAddr = functionSymbol.address();
+		MethodHandle mh = clinker.downcallHandle(memAddr, mt, fd);
+		short result = (short)mh.invokeExact((short)24, (short)32);
+		Assert.assertEquals(result, (short)56);
+	}
+	
+	@Test
+	public void test_addTwoInts() throws Throwable {
+		MethodType mt = MethodType.methodType(int.class, int.class, int.class);
+		FunctionDescriptor fd = FunctionDescriptor.of(C_INT, C_INT, C_INT);
+		Symbol functionSymbol = nativeLib.lookup("add2Ints").get();
+		MethodHandle mh = clinker.downcallHandle(functionSymbol, mt, fd);
+		int result = (int)mh.invokeExact(112, 123);
+		Assert.assertEquals(result, 235);
+	}
+	
+	@Test
+	public void test_addTwoNegtiveInts() throws Throwable {
+		MethodType mt = MethodType.methodType(int.class, int.class, int.class);
+		FunctionDescriptor fd = FunctionDescriptor.of(C_INT, C_INT, C_INT);
+		Symbol functionSymbol = nativeLib.lookup("add2Ints").get();
+		MethodHandle mh = clinker.downcallHandle(functionSymbol, mt, fd);
+		int result = (int)mh.invokeExact(-112, -123);
+		Assert.assertEquals(result, -235);
+	}
+	
+	@Test
+	public void test_addIntAndIntFromPointer() throws Throwable {
+		MethodType mt = MethodType.methodType(int.class, int.class, MemoryAddress.class);
+		FunctionDescriptor fd = FunctionDescriptor.of(C_INT, C_INT, C_POINTER);
+		Symbol functionSymbol = nativeLib.lookup("addIntAndIntFromPointer").get();
+		MethodHandle mh = clinker.downcallHandle(functionSymbol, mt, fd);
+		
+		MemorySegment intSegmt = MemorySegment.allocateNative(C_INT);
+		MemoryAccess.setInt(intSegmt, 215);
+		int result = (int)mh.invokeExact(321, intSegmt.address());
+		intSegmt.close();
+		Assert.assertEquals(result, 536);
+	}
+	
+	@Test
+	public void test_addTwoInts_fromMemAddr() throws Throwable {
+		MethodType mt = MethodType.methodType(int.class, int.class, int.class);
+		FunctionDescriptor fd = FunctionDescriptor.of(C_INT, C_INT, C_INT);
+		Symbol functionSymbol = nativeLib.lookup("add2Ints").get();
+		MemoryAddress memAddr = functionSymbol.address();
+		MethodHandle mh = clinker.downcallHandle(memAddr, mt, fd);
+		int result = (int)mh.invokeExact(112, 123);
+		Assert.assertEquals(result, 235);
+	}
+	
+	@Test
+	public void test_addIntsWithVaList() throws Throwable {
+		Symbol functionSymbol = nativeLib.lookup("addIntsFromVaList").get();
+		MethodType mt = MethodType.methodType(int.class, int.class, VaList.class); 
+		FunctionDescriptor fd = FunctionDescriptor.of(C_INT, C_INT, C_VA_LIST);
+		NativeScope nativeScope = NativeScope.unboundedScope();
+		VaList vaList = CLinker.VaList.make(vaListBuilder -> vaListBuilder.vargFromInt(C_INT, 700)
+															.vargFromInt(C_INT, 800)
+															.vargFromInt(C_INT, 900)
+															.vargFromInt(C_INT, 1000),
+															nativeScope);
```

Comment:
Excessive indentation.

Reply:
Fixed at all places.

Reply:
I think the line-breaks made for easier reading. I'd like to seem them return, but don't indent more than one tab extra compared with the start of the declaration:
```
		VaList vaList = CLinker.VaList.make(vaListBuilder -> vaListBuilder
			.vargFromInt(C_INT, 700)
			.vargFromInt(C_INT, 800)
			.vargFromInt(C_INT, 900)
			.vargFromInt(C_INT, 1000), nativeScope);
```

Reply:
Fixed as suggested above.